.github/workflows/test_branches.yml

```diff
@@ -868,13 +872,18 @@ jobs:
 
     - name: Install Pyomo
       run: |
+        # Turn on cython through env vars
+        if [[ "${{ matrix.other }}" == *cython* ]]; then
+            echo "Cython requested; turning on cythonization options in environment"
+            export PYOMO_SETUP_ARGS=--with-cython
+        fi
```

Comment:
Same question as above...

Reply:
Because the only reason we use `setup_options` is for cythonization. I don't really care either way, but it seemed redundant, so tell me what you'd prefer.